src/store/reducer/board.ts

```diff
@@ -75,6 +83,12 @@ export const boardReducer = (state: BoardState = {status: "unknown"}, action: Re
       };
     }
     case ActionType.UpdatedBoard: {
+      // User notification
+      if (state.data?.voting !== action.board.voting && action.board.voting !== undefined) {
+        if (action.board.voting === "active") Toast.success("Voting phase started! You can vote now!");
```

Comment:
Imo you should always use parentheses, even if they're not strictly required, to have code that's clear and correct. 

Parentheses make code easier to understand and help reducing silly mistakes

Reply:
Yes you are right. Autoformater changes this to single lines, which makes it unreadable. Parantheses added.